docs/QbeastFormat.md

```diff
@@ -63,15 +63,15 @@ Here you can see the changes on the `AddFile` **`tags`** information
 }
 ```
 
-| Term           | Description                                                                                                                                  |
-|----------------|----------------------------------------------------------------------------------------------------------------------------------------------|
-| `revision`     | The metadata of the tree. Locates the particular tree in the space, which includes the min/max values for each column indexed and timestamp. |
-| `blocks`       | A list of the blocks that are stored inside the file. Can belong to different cubes.                                                         |
-| `cube`         | The serialized representation of the Cube.                                                                                                   |
-| `minWeight`    | The minimum weight of the block.                                                                                                             |
-| `maxWeight`    | The maximum weight of the block.                                                                                                             |
-| `elementCount` | The number of elements in the block.                                                                                                         |
-| `replicated`   | A flag that indicates if the block is replicated. Replication is **not available** from v0.6.0.                                                  |
+| Term           | Description                                                                                                    |
+|----------------|----------------------------------------------------------------------------------------------------------------|
+| `revision`     | **Locates a particular `min`-`max` range of the columns** in the space, and includes a Timestamp of the creation time. |
+| `blocks`       | A **list of the blocks** that are stored inside the file. Can belong to different cubes.                           |
+| `cube`         | The **serialized representation of the Cube**.                                                                     |
```

Comment:
I don't understand what serialized means. Is it a hash ID or something similar?

Reply:
I would change it for "the string representation of..."